### PR TITLE
continue on empty strings

### PR DESCRIPTION
**Pull request by huttmf**
Addresses Bug#14

### CODE REVIEW

File: plugins/inputs/aos/aos.go [hunk 1]
```diff
@@ -103,8 +103,9 @@ func (ssl *streamAos) extractAlertData(alertType string, tags map[string]string,
 		}
 		field_name := propDataType.Prop[i].OrigName
 
-		// Skip field with XXX_
-		if strings.Contains(field_name, "XXX_") {	continue }
+		// Skip field with XXX_ and empty strings
+		if (field_name == "") { continue }
```
Comment (Ragsboss) [inline plugins/inputs/aos/aos.go:107]: When can the field_name be empty? Is this indicative of a mismatched proto schema?

File: plugins/inputs/aos/aos_test.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    "github.com/influxdata/telegraf/plugins/inputs/aos"
+    "github.com/influxdata/telegraf/plugins/inputs/aos/aos_streaming"
+    "github.com/influxdata/telegraf/testutil"
+    "github.com/stretchr/testify/assert"
+)
+
+var aos_server string = os.Getenv("AOS_SERVER")
+
+func TestExtractAlertDataForProbeAlert(t *testing.T) {
```
Comment (Ragsboss) [inline plugins/inputs/aos/aos_test.go:15]: I'd suggest renaming this method to make it more specific and reflect the particular aspect it is validating. Ex: `TestExtractAlertDataForIntegerProbeAlert`.

Comment (Ragsboss) [inline plugins/inputs/aos/aos_test.go:15]: BTW is this test exercising the issue mentioned in issue #14 ?

**Review by Ragsboss — CHANGES_REQUESTED** [review]

File: plugins/inputs/aos/aos.go (hunk 1 above)
Comment (huttmf) [reply plugins/inputs/aos/aos.go:107]: I don't think this a problem. This issue is most likely a duplicate of #10. Pulling the latest from apstra/telefraf does not exhibit the problem.

File: plugins/inputs/aos/aos_test.go (hunk 2 above)
Comment (huttmf) [reply plugins/inputs/aos/aos_test.go:15]: Yes - will continue improving test coverage.

File: plugins/inputs/aos/aos.go (hunk 1 above)
Comment (Ragsboss) [reply plugins/inputs/aos/aos.go:107]: I'm confused then. Why do we need this change/pr in this case?

Comment (huttmf) [reply plugins/inputs/aos/aos.go:107]: We don't need this particular change - but I think we should go ahead with the PR for the testing support.

Comment (Ragsboss) [reply plugins/inputs/aos/aos.go:107]: Can we revert this change given it's not needed..

File: plugins/inputs/aos/aos_test.go (hunk 2 above)
Comment (Ragsboss) [reply plugins/inputs/aos/aos_test.go:15]: I meant to say rename existing test method name..

**Pull request closed without merging**